### PR TITLE
fix(DB/SAI): Wastewalker Slave&Wastewalker Worker

### PR DESCRIPTION
**Pull request by Selyria**
My first self-written fix.

Error: If you attack the groups the Coilfang slavehandler gets help from 3 creatures you kill the Coilfang slavehandler the 3 creatures run away. 
Problem was that they didn't run away but still attacked a player until they despawned after 5 seconds. 

Closes nothing

**TESTS PERFORMED:**
I tested it myself and it worked.
Fix is not on the server https://youtu.be/AVlEbBvJiOk
Fix on Server https://youtu.be/AVlEbBvJiOk?t=41

**HOW TO TEST THE CHANGES:**
Go and attack Coilfang slavehandler there and kill him the helpers now go away and despawn.

.go -36.88 -9.77 -1.5 547

I hope it's all correct now.

Target branch(es):
Master

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
@Selyria  No newline at the end of file is a git warning for windows users.

Btw have you run this script on your azerothcore local repository  https://github.com/azerothcore/azerothcore-wotlk/tree/master/apps/git_tools ?

And thank you for that PR!

**Comment by FrancescoBorzi** [comment]
good job @Selyria 

waiting for tester feedback

**Comment by Selyria** [comment]
ah okay good to know with the newline i can prevent this message? who yes how.

https://github.com/azerothcore/azerothcore-wotlk/tree/master/apps/git_tools ? no i have not used have stuck to the wiki 

> @Selyria No newline at the end of file is a git warning for windows users.
> 
> Btw have you run this script on your azerothcore local repository https://github.com/azerothcore/azerothcore-wotlk/tree/master/apps/git_tools ?
> 
> And thank you for that PR!

ah okay good to know with the newline i can prevent this message? who yes how.

https://github.com/azerothcore/azerothcore-wotlk/tree/master/apps/git_tools ? no i have not used have stuck to the wiki

**Comment by BarbzYHOOL** [comment]
Ok, can you run this script then? with bash or "git bash", it will help you with GIT when writing commits.

To prevent the message for git on windows, you should configure your text editor to automatically add a new line at the end of the file.

**Comment by deadmanevil** [comment]
if you already tested on in source and if the npc is no longer returned to attack again when the sklaventreiber is killed

**Comment by Mindustry** [comment]
Tested this on a Windows 10 64-bit build.
Found everything to be working properly after applying @Selyria's patch. Good find and fix!
snipped
Here's before applying the SQL
[Without Fix](https://webmshare.com/Ry5eE)

And here is the update working successfully.
[With Fix](https://webmshare.com/LxzjN)

File: data/sql/updates/pending_db_world/rev_1558744539335597800.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+(17963, 0, 3, 0, 0, 0, 100, 2, 1300, 3700, 5000, 8000, 0, 11, 15497, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - In Combat - Cast Frostbolt'),
+(17963, 0, 4, 0, 0, 0, 100, 4, 1300, 3700, 5000, 8000, 0, 11, 12675, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - In Combat - Cast Frostbolt'),
+(17963, 0, 5, 6, 38, 0, 100, 0, 1, 1, 0, 0, 0, 69, 1, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 63.3, -58.76, -1.5, 0, 'Wastewalker Slave - On Data Set - Move Point'),
+(17963, 0, 6, 7, 61, 0, 100, 0, 0, 0, 0, 0, 0, 41, 5000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - On Data Set - Despawn'),
+(17963, 0, 7, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - On Data Set - Say Line 0'),
+(17963, 0, 8, 0, 38, 0, 100, 0, 2, 2, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 21, 30, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - On Data Set - Attack Start'),
+(17963, 0, 9, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 27, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Slavehandler Death Damage Stop to Player');
```
Comment (Stoabrogga) [inline data/sql/updates/pending_db_world/rev_1558744539335597800.sql]: Please update the comment to reflect the correct NPC, event and action.
```suggestion
(17963, 0, 9, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 27, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Slave - On Data Set - Stop Combat');
```

File: data/sql/updates/pending_db_world/rev_1558744539335597800.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+(17964, 0, 0, 0, 0, 0, 100, 2, 2300, 7700, 9000, 11000, 0, 11, 13738, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - In Combat - Cast Rend'),
+(17964, 0, 1, 0, 0, 0, 100, 4, 2300, 7700, 9000, 11000, 0, 11, 37662, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - In Combat - Cast Rend'),
+(17964, 0, 2, 3, 38, 0, 100, 0, 1, 1, 0, 0, 0, 69, 1, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 63.3, -58.76, -1.5, 0, 'Wastewalker Worker - On Data Set - Move Point'),
+(17964, 0, 3, 4, 61, 0, 100, 0, 0, 0, 0, 0, 0, 41, 5000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - On Data Set - Despawn'),
+(17964, 0, 4, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - On Data Set - Say Line 0'),
+(17964, 0, 5, 0, 38, 0, 100, 0, 2, 2, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 21, 30, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - On Data Set - Attack Start'),
+(17964, 0, 6, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 27, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Slavehandler Death Damage Stop to Player');
```
Comment (Stoabrogga) [inline data/sql/updates/pending_db_world/rev_1558744539335597800.sql]: Please update the comment to reflect the correct NPC, event and action.
```suggestion
(17964, 0, 6, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 27, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Wastewalker Worker - On Data Set - Stop Combat');
```

**Comment by BarbzYHOOL** [comment]
Selyria, you should join us on discord (you have already ranks there)

And thank you @Mindustry for testing

**Comment by Selyria** [comment]
> Selyria, you should join us on discord (you have already ranks there)

I'm already inside with you only speak almost no English

And I travel a lot with my Ts3 server. Otherwise work just ^^

without deepl.com, I would be on .... Nice translators

**Pull request merged**